pegasus/sites.v3/code.org/views/learn_carousels.haml

```diff
@@ -1,4 +1,6 @@
-- categories = [{tag:"JavaScript", name: I18n.t(:carousel_heading_javascript)}, {tag:"Beginner", name: I18n.t(:carousel_heading_beginners)}, {tag:"Unplugged", name: I18n.t(:carousel_heading_unplugged)}, {tag:"Apps", name: I18n.t(:carousel_heading_devices)}, {tag:"OtherLang", name: I18n.t(:carousel_heading_languages)}, {tag: "Univ", name: I18n.t(:carousel_heading_university)}, {tag: "Web", name: I18n.t(:carousel_heading_webpages)}, {tag: "Robot", name: I18n.t(:carousel_heading_robots)}, {tag: "IDE", name: I18n.t(:carousel_heading_ide)}]
+- learn_categories = [{tag:"JavaScript", name: I18n.t(:carousel_heading_javascript)}, {tag:"Beginner", name: I18n.t(:carousel_heading_beginners)}, {tag:"Unplugged", name: I18n.t(:carousel_heading_unplugged)}, {tag:"Apps", name: I18n.t(:carousel_heading_devices)}, {tag:"OtherLang", name: I18n.t(:carousel_heading_languages)}, {tag: "Univ", name: I18n.t(:carousel_heading_university)}, {tag: "Web", name: I18n.t(:carousel_heading_webpages)}, {tag: "Robot", name: I18n.t(:carousel_heading_robots)}, {tag: "IDE", name: I18n.t(:carousel_heading_ide)}]
```

Comment:
I think this would be more readable with newlines.  You can create an array and then add each entry with <<.  See inspirational_videos.haml for an exmaple.